helm/kubernetes-namespace-prep/templates/conjur-connect-configmap.yml

```diff
@@ -0,0 +1,21 @@
+---
+{{- if .Values.conjurConfigMap.create -}}
+{{- $g := (lookup "v1" "ConfigMap" .Values.authnk8s.namespace .Values.authnk8s.goldenConfigmap).data -}}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: conjur-connect-configmap 
+  labels: 
+    app.kubernetes.io/name: {{ .Values.authnk8s.namespace }}-authn-k8s
```

Comment:
Same comment as above, I think Issue requirements aren't quite right... and the proper standard Kubernetes labels for this ConfigMap should be something like this:

```
    app.kubernetes.io/name: "conjur-conn-configmap"
    app.kubernetes.io/component: "conjur-conn-config"
    app.kubernetes.io/instance: {{ .Release.Namespace }}
    app.kubernetes.io/part-of: "authn-k8s-namespace-config"
```